src/main/java/ti4/service/unit/ParseUnitService.java

```diff
@@ -64,6 +64,7 @@ private ParsedUnit parseUnit(String unitListToken, String color, Tile tile, Gene
         String originalUnit;
         if (NumberUtils.isDigits(firstToken)) {
             count = Math.max(Integer.parseInt(firstToken), 1);
+            if (!unitInfoTokenizer.hasMoreTokens()) return null;
```

Comment:
People goof up the input format in a variety of ways. Help with this one: https://discord.com/channels/943410040369479690/1341438476528123904/1376549368751980656